stubs/PyAutoGUI/pyautogui/__init__.pyi

```diff
@@ -70,10 +74,10 @@ def size() -> Size: ...
 
 resolution = size
 
-def onScreen(x: _NormalizeableXArg | None, y: SupportsInt | None = None) -> bool: ...
+def onScreen(x: _NormalizeableXArg | None, y: ConvertibleToInt | None = None) -> bool: ...
```

Comment:
Are there any examples in the documentation where `y` is called with a `str` argument? Or, is there any particular reason that you know of to pass a `str` to the `y` parameter?

It's true that the function at runtime calls `int()` on the argument passed to `y` before doing anything with it, so if you pass `"10"` to the `y` parameter, it _will work_. But what's the reason for doing so? `SupportsInt` isn't a perfect type hint here, but it ensures that the type checker will still emit an error if a user accidentally tries to pass `"foo"` to the `y` parameter -- and with this change, it wouldn't.

_In general_, typeshed tries to put more weight on avoiding false positives than on avoiding false negatives. But in a situation like this, where (as far as I can see) there's no real use case for passing a `str`, I think we can and should be more opinionated than the function at runtime.

Reply:
I don't think so. I'll revert here. How about the normalizable X arg (see `_NormalizeableXArg` aslias)? It can be a `str`, but `str` there means "filename on screen"

Reply:
Yeah, I'm fine with that one!